arch/arm/dts/imx7ulpea-ucom-kit_v2.dts

```diff
@@ -69,7 +69,6 @@
 		};
 
 	};
-
```

Comment:
Can we leave this line in place?

Reply:
I'll remove the dts changes. lets merge yours instead